spec/02-integration/03-dao/03-crud_spec.lua

```diff
@@ -604,6 +634,20 @@ helpers.for_each_dao(function(kong_config)
           assert.falsy(err)
           assert.same(api_fixture, api)
         end)
+        it("unset ngx.null fields", function()
```

Comment:
Because the previous test unsets `uris` already, we don't know if this test succeeds because of the newly introduced code or because the previous test is already passing. Could we rework the logic a bit here? A suggestion would be to introduce the following in the previous test:

```lua
it("unset nil fields", function()
  finally(function()
    -- re add the `uris` field
    api_fixture.uris = "/mockbin"
    assert(api:update(api_fixture, api_fixture, { full = true}))
  end)

  -- rest of the test
end)

it("unset ngx.null fields", function()
  -- ...
end)
```